### PR TITLE
Update submodules

### LINKED ISSUE #29: Checkout the v1.0.0 tag on `morpho-blue`

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Fixes #29

### CODE REVIEW

File: lib/forge-std
Comment (Rubilmax) [inline lib/forge-std:1]: so we keep it?

**Review by Rubilmax — APPROVED** [review]

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]
I think it points to main but perhaps it's better to point to the correct release v1.0.0, wdyt?

File: lib/forge-std
Comment (Jean-Grimal) [reply lib/forge-std:1]: Forgot to remove it

**Review by MerlinEgalite — COMMENTED** [review]
Oops just wanted to make a comment

**Review by Rubilmax — APPROVED** [review]

**Review by Jean-Grimal — COMMENTED** [review]
LGTM

**Review by MerlinEgalite — APPROVED** [review]

**Pull request merged**